noria-server/dataflow/src/node/special/reader.rs

```diff
@@ -35,7 +35,7 @@ pub struct Reader {
     streamers: Vec<channel::StreamSender<Vec<StreamUpdate>>>,
 
     for_node: NodeIndex,
-    replica_index: Option<usize>,
+    reader_index: usize,
```

Comment:
What does this field mean? Is it the index of this `Reader` in the list of `Reader`s for its ancestor?

Reply:
Yes, added a comment.